project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.2.7
\ No newline at end of file
+sbt.version = 1.3.0
```

Comment:
sbt 1.3.0 is broken. You should use 1.3.2

Reply:
just curious what's wrong with `1.3.0`?
and. there is already a `1.3.3` one.. looks fine, right?

Reply:
https://discuss.lightbend.com/t/announce-sbt-1-3-2/5111
I saw the 1.3.3 announcement ½ hour after I created the review :)